localization/yabloc/yabloc_particle_filter/src/camera_corrector/camera_particle_corrector_core.cpp

```diff
@@ -147,10 +147,10 @@ void CameraParticleCorrector::on_line_segments(const PointCloud2 & line_segments
   ParticleArray weighted_particles = opt_array.value();
 
   bool publish_weighted_particles = true;
-  const Pose meaned_pose = mean_pose(weighted_particles);
+  const Pose mean_pose = mean_pose(weighted_particles);
```

Comment:
I guess this will be compilation error?
```suggestion
  const Pose mean_pose_msg = mean_pose(weighted_particles);
```

There are  variables with the same name in gnss_particle_corrector_core,cpp.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/4281/commits/90e3134477b80dad913c865cacdaa0bc9985a724
Reflected here. waiting for CI to pass